app/assets/scripts/views/location/temporal-measurements.js

```diff
@@ -0,0 +1,213 @@
+      setDayCoverage(parseDay(tabMeasurements));
+      setMonthCoverage(parseMonth(tabMeasurements));
+    }
+  }, [fetched, activeTab]);
+
+  if (!fetched && !fetching) {
+    return null;
+  }
+
+  const parseHour = measurements => {
+    const hourCount = measurements.reduce((prev, curr) => {
+      const hour = new Date(curr.date.local).getHours();
+      prev[hour] = prev[hour] ? prev[hour] + 1 : 1;
+      return prev;
+    }, {});
+    const orderedHoursCount = {};
+    Object.keys(hourCount)
+      .sort()
+      .forEach(function (key) {
+        orderedHoursCount[key] = hourCount[key];
+      });
+    return orderedHoursCount;
+  };
+
+  const parseDay = measurements => {
+    const daysOfWeek = ['SUN', 'MON', 'TUES', 'WED', 'THURS', 'FRI', 'SAT'];
+    const dayCount = measurements.reduce((prev, curr) => {
+      const day = daysOfWeek[new Date(curr.date.local).getDay()];
+      prev[day] = prev[day] ? prev[day] + 1 : 1;
+      return prev;
+    }, {});
+    let sortedDayCount = {};
+    daysOfWeek.forEach(day =>
+      dayCount[day]
+        ? (sortedDayCount[day] = dayCount[day])
+        : (sortedDayCount[day] = 0)
+    );
+    return sortedDayCount;
+  };
+
+  const parseMonth = measurements => {
+    const monthOfYear = [
+      'JAN',
+      'FEB',
+      'MAR',
+      'APR',
+      'MAY',
+      'JUNE',
+      'JUL',
+      'AUG',
+      'SEP',
+      'OCT',
+      'NOV',
+      'DEC',
+    ];
+    const monthCount = measurements.reduce((prev, curr) => {
+      const month = monthOfYear[new Date(curr.date.local).getMonth()];
+      prev[month] = prev[month] ? prev[month] + 1 : 1;
+      return prev;
+    }, {});
+    let sortedMonthCount = {};
+    monthOfYear.forEach(month =>
+      monthCount[month]
+        ? (sortedMonthCount[month] = monthCount[month])
+        : (sortedMonthCount[month] = 0)
+    );
+    return sortedMonthCount;
+  };
+
+  if (fetching) {
+    return <LoadingMessage />;
+  } else if (error) {
+    return (
+      <ErrorMessage>
+        <p>We couldn&apos;t get any data.</p>
+        <InfoMessage>
+          <p>Please try again later.</p>
+          <p>
+            If you think there&apos;s a problem, please{' '}
+            <a href="mailto:info@openaq.org" title="Contact openaq">
+              contact us.
+            </a>
+          </p>
+        </InfoMessage>
+      </ErrorMessage>
+    );
+  }
+
+  return (
+    <Card
+      gridColumn={'1  / -1'}
+      renderHeader={() => (
+        <CardHeader className="card__header">
+          <TabbedSelector
+            tabs={parameters}
+            activeTab={activeTab}
+            onTabSelect={t => {
+              setActiveTab(t);
+            }}
+          />
+          <CardTitle>Temporal Coverage</CardTitle>
+        </CardHeader>
+      )}
+      renderBody={() => (
+        <div className="card__body">
+          {activeTabMeasurments.length ? (
+            <CardList gridTemplateColumns={'repeat(3, 1fr)'} className="inner">
+              <Card
```

Comment:
Should each chart be a Card within the card? Seems odd to me... I'd think the Card component has to many features we don't need here at all. Just a div could do?

Reply:
I agree